package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "security-blogs",
-  "version": "3.1.2",
+  "version": "3.1.3",
   "main": "./integration.js",
   "private": true,
   "dependencies": {
     "async": "^3.2.4",
```

Comment:
@CJ-Polarity I think there is a 3.2.5 for async now.  Can you bump this and update the package-lock

```
"async": "^3.2.5"
```